examples/vite-react-with-typescript/src/App.tsx

```diff
@@ -0,0 +1,264 @@
+import { ReactNode, useEffect, useState } from 'react';
+import './App.css';
+import {
+  Provider as MediaChromeProvider,
+  useMediaDispatch as useDispatch,
+  useFullscreenRefCallback,
+  useMediaRefCallback,
+  useMediaSelector as useSelector,
+} from 'media-chrome/react/media-store';
+import { constants } from 'media-chrome';
+const { MediaUIEvents } = constants;
+
+const PlayButton = () => {
```

Comment:
This is analagous to the next.js example of using the new react context+provider+hooks integrating with `MediaStore`